### PR TITLE
Admin navigation #136

### PR DESCRIPTION
**Pull request by nicovigil1**
adds admin navigation bar - also adds routes and admin users controller with a rudimentary start on the admin users index page

### CODE REVIEW

**Comment by nicovigil1** [comment]
closes #136

File: app/views/_nav_bar.html.erb [hunk 1]
```diff
@@ -2,19 +2,26 @@
   <%= link_to "home", root_path  %>
   <%= link_to "browse items", items_path  %>
   <%= link_to "browse merchants", merchants_path  %>
-  <%= link_to "shopping cart", cart_path  %>
 
-  <% if current_user && current_user.role == "default" %>
+  <% unless current_user && current_user.role == "admin" %>
+    <%= link_to "shopping cart", cart_path  %>
+  <% end %>
+  <% if current_user && current_user.role = "admin"  %>
```
Comment (tonyr729) [inline app/views/_nav_bar.html.erb]: Line  `current_user.role == "admin"`

File: app/views/_nav_bar.html.erb [hunk 2]
```diff
@@ -2,19 +2,26 @@
   <%= link_to "home", root_path  %>
   <%= link_to "browse items", items_path  %>
   <%= link_to "browse merchants", merchants_path  %>
-  <%= link_to "shopping cart", cart_path  %>
 
-  <% if current_user && current_user.role == "default" %>
+  <% unless current_user && current_user.role == "admin" %>
+    <%= link_to "shopping cart", cart_path  %>
```
Comment (tonyr729) [inline app/views/_nav_bar.html.erb]: You can include this into the conditional below in an `else` statement

File: app/views/admin/users/index.html.erb [hunk 3]
```diff
@@ -0,0 +1,30 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>All da Users</title>
+</head>
+<body>
+    <table>
```
Comment (tonyr729) [inline app/views/admin/users/index.html.erb]: The index view of users will be rendered into the `yield` of our application.html.erb so we do not want to include the full HTML and header just the content to be displayed in the body.

File: app/views/_nav_bar.html.erb (hunk 2 above)
Comment (nicovigil1) [reply app/views/_nav_bar.html.erb]: changed

File: app/views/_nav_bar.html.erb (hunk 1 above)
Comment (nicovigil1) [reply app/views/_nav_bar.html.erb]: good catch

**Comment by nicovigil1** [comment]
fixed changes and pushed!

File: app/views/admin/users/index.html.erb (hunk 3 above)
Comment (nicovigil1) [reply app/views/admin/users/index.html.erb]: just fixed!

**Pull request merged**